ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts

```diff
@@ -92,4 +93,11 @@ export class ActivityServiceAgent extends UserServiceAgent<any> {
     return this.patchObservable(`/studies/${studyGuid}/activities/${activityGuid}`, payload).pipe(
       map(httpResponse => httpResponse));
     }
+
+    public getAnswerByQuestionStableId(
+      studyGuid: string,
+      questionStableId: string
+    ): Observable<ActivityAnswerResponse> {
+        return this.getObservable(`/studies/${studyGuid}/answers/${questionStableId}`);
```

Comment:
As I mentioned in the other PR's [comment](https://github.com/broadinstitute/ddp-study-server/pull/463#discussion_r513585969), how about we make the endpoint `/activities/${activityGuid}/answers/${questionStableId}`?

Reply:
Alright, we decided to remove this endpoint and therefore this change is not needed.